serializers/postgres.rb

```diff
@@ -7,8 +7,10 @@ def self.serialize_internal(pg, options = {})
       name: pg.name,
       state: pg.display_state,
       location: pg.display_location,
-      vm_size: pg.target_vm_size,
-      storage_size_gib: pg.target_storage_size_gib,
+      vm_size: pg.representative_server&.vm&.display_size,
```

Comment:
I think we should default to `target_vm_size` if this is `nil`. Same for `storage_size_gib`